src/main/java/edu/escuelaing/ieti/matchmaking/AppConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package edu.escuelaing.ieti.matchmaking;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.MongoDbFactory;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.SimpleMongoDbFactory;
+
+@Configuration
+public class AppConfiguration {
+
+    @Bean
+    public MongoDbFactory mongoDbFactory() throws Exception {
+
+        MongoClientURI uri = new MongoClientURI(
+                "mongodb+srv://admin:admin123@proyecto-puxek.mongodb.net/test?retryWrites=true&w=majority");
+
+        MongoClient mongoClient = new MongoClient(uri);
+
+        return new SimpleMongoDbFactory(mongoClient, "test");
+    }
+
+    @Bean
+    public MongoTemplate mongoTemplate() throws Exception {
```

Comment:
The mongo template is only needed if you are planning to make custom queries. Consider leveraging the Spring Data framework queries instead